guides/common/modules/proc_monitoring-puppet.adoc

```diff
@@ -3,24 +3,26 @@
 [id="Monitoring_Puppet_{context}"]
 = Monitoring Puppet
 
-In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and click the name of the selected {SmartProxy}.
-At the *Puppet* tab you can find the following:
-
-* A summary of Puppet events, an overview of latest Puppet runs, and the synchronization status of associated hosts at the *General* sub-tab.
-* A list of Puppet environments at the *Environments* sub-tab.
-
-At the *Puppet CA* tab you can find the following:
-
-* A certificate status overview and the number of autosign entries at the *General* sub-tab.
-* A table of CA certificates associated with the {SmartProxy} at the *Certificates* sub-tab.
-Here you can inspect the certificate expiry data, or cancel the certificate by clicking *Revoke*.
-* A list of autosign entries at the *Autosign entries* sub-tab.
-Here you can create an entry by clicking *New* or delete one by clicking *Delete*.
+[role="_abstract"]
+You can monitor Puppet activity and certificate management through the Smart Proxy interface.
+This helps you verify that Puppet services run correctly and troubleshoot any issues.
```

Comment:
IMO it does not. It only shows the current state of Puppet, aka. Puppet envs and class per env.

If you turn off `puppet` and `puppetserver` services on Smart Proxies, the WebUI will still show them but also show an error message:

````
Failure: ERF50-5345 [Foreman::WrappedException]: Unable to connect ([ProxyAPI::ProxyException]: ERF12-5356 [ProxyAPI::ProxyException]: Unable to get PuppetCA certificates ([RestClient::NotAcceptable]: 406 Not Acceptable) for proxy https://foreman.example.com:9090/puppet/ca)
````

It will not show any information about running services. I suggest to drop line 8:

```suggestion
```